pycbc/events/stat.py

```diff
@@ -2084,6 +2090,41 @@ def lognoiserate(self, trigs):
         return logr_n
 
 
+class DQExpFitFgBgKDEStatistic(DQExpFitFgBgNormStatistic):
+    """
+    The ExpFitFgBgKDEStatistic with DQ-based reranking.
+
+    This is the same as the DQExpFitFgBgNormStatistic except the signal
+    rate is adjusted according to the KDE statistic files
+    """
+
+    def __init__(self, sngl_ranking, files=None, ifos=None, **kwargs):
+        """
+        Inherited from DQExpFitFgBgNormStatistic, see docstring there
+        but with find_kdes fucntion from ExpFitFgBgKDEStatistic added
```

Comment:
Not sure we want to squash this docstring if it is not a straight inheritance without change .. probably better to keep as it was.